projects/core/src/lib/authorization/components/role-select/role-select.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="jsmu-role-select">
+	<div class="main-div">
+		<span class="main-div__text">Please select your role below</span>
+		<div class="roles">
+			<jsmu-role
+				*ngFor="let roleInfo of rolesInfo"
+				[role]="roleInfo"
+				(roleEvent)="switchRole($event)"
+			></jsmu-role>
+		</div>
+	</div>
+</div>
```

Comment:
rewrite to the BEM one block jsmu-role-select with elements __main, __text, __roles

Reply:
Done